server/data/data-cassandra/src/main/java/org/apache/james/domainlist/cassandra/CassandraDomainList.java

```diff
@@ -100,7 +100,7 @@ private PreparedStatement prepareReadAllStatement(Session session) {
     }
 
     @Override
-    public boolean containsDomain(String domain) throws DomainListException {
+    public boolean containsDomainInternal(String domain) throws DomainListException {
```

Comment:
do you really want to change the visibility from protected to public? (same question in others implementations)